### ISSUE TITLE
Error in Acrobat Reader DC when using dynamic: true on repeater.

### ISSUE THREAD

**Issue by vanboom** [prawnpdf/prawn#925]
Similar to #497 
Using Prawn 2.0.2 in a Ruby on Rails application...

I have defined a stamp using create_stamp that includes an svg image and a table.  
```ruby
pdf.create_stamp("mystamp") do
  ... render a svg image
  ... render a table
  pdf.stroke_bounds
end
```
Then, called the stamp for each page at the bottom of my pdf 
``` ruby
pdf.repeat((s..e).to_a, :dynamic=>true) do
	pdf.stamp logbook_entry.id.to_s
end
```
This induces the same error in Acrobat Reader as found in issue #497 

Setting dynamic=>false causes the PDF to render perfectly in Acrobat Reader DC.

**Comment by iamjohnford** [comment]
If you provide a full working example it would be easier for other people to test.

**Comment by packetmonkey** [comment]
Yes, if you can please post a full code snippet to generate the bad PDF we can check it out.
